modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/schema/IgniteIndex.java

```diff
@@ -59,6 +59,11 @@ public RelCollation collation() {
         return collation;
     }
 
+    /** */
+    public boolean rewindable() {
```

Comment:
i surprised, what does it mean in case of class usage ? Also no inheritance found why do we need such a property here ?

Reply:
To make it similar to the IgniteTable interface. You could override this in tests if needed.

Reply:
in private discussion @zstan have convinced me to get rid of this property